tcms/static/js/testrun_actions.js

```diff
@@ -246,6 +255,7 @@ Nitrate.TestRuns.AssignCase.on_load = function() {
 
 
 function updateRunStatus(object_pk, value, callback) {
+  // TODO: replace this view with an API call
```

Comment:
I thought this was done together with the HTML changes in this part of the page but apparently not. See how bulk-add-comments works, it should be fairly similar. IMO you can add into this PR since it's related and it would be the smallest if all changes. And remove everything we don't need afterwards.

Reply:
it is not, because we don't have a bulk update API. see my comment above. do you want to introduce this API as part of this PR?

Reply:
you don't need a bulk-update API. You can loop over all of the selected rows and send RPC calls for every one of them. B/c at the end the entire page will be refreshed this needs to be executed synchronously. No ideal but is an acceptable compromise. In either case if you are going to upgrade hundreds of cases at once where the sync nature of execution becomes a problem then you are having other types of issues more likely.